### PR TITLE
desktop: app kickstart

### PR DESCRIPTION
**Pull request by zoubingwu**
todo
- auto open devtool on start
- react/reframe devtools
- electron packager

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```
Comment (theJian) [inline .editorconfig]: I would like to leave `trim_trailing_whitespace` to `false` for `.md` files.

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
-# cheerybin.desktop
\ No newline at end of file
+# cheerybin.desktop
+
+## Development
+
+```sh
+yarn
```
Comment (theJian) [inline README.md]: I propose switching back to npm. For [speed](https://blog.npmjs.org/post/171556855892/introducing-npm-ci-for-faster-more-reliable) and [security](https://blog.npmjs.org/post/173719309445/npm-audit-identify-and-fix-insecure).

File: package.json [hunk 3]
```diff
@@ -15,5 +20,15 @@
...
     "url": "https://github.com/cheerybin/cheerybin.desktop/issues"
   },
-  "homepage": "https://github.com/cheerybin/cheerybin.desktop#readme"
+  "homepage": "https://github.com/cheerybin/cheerybin.desktop#readme",
+  "devDependencies": {
+    "electron-prebuilt": "^1.4.13",
+    "shadow-cljs": "^2.6.6"
+  },
+  "dependencies": {
+    "create-react-class": "^15.6.3",
```
Comment (theJian) [inline package.json:29]: What was this used for?

File: shadow-cljs.edn [hunk 4]
```diff
@@ -0,0 +1,20 @@
+;; shadow-cljs configuration
+{:source-paths ["src"]
+
+ :dependencies [[org.clojure/clojure "1.8.0"]
```
Comment (theJian) [inline shadow-cljs.edn]: Could we use the latest version of clojure? Which is 1.9.

File: package.json (hunk 3 above)
Comment (zoubingwu) [reply package.json:29]: reagent's dependencies

**Comment by zoubingwu** [comment]
poke @theJian

**Pull request merged**